Core.ApplicationServices/Organizations/IOrganizationService.cs

```diff
@@ -49,5 +49,7 @@ public interface IOrganizationService
         Result<IEnumerable<Organization>, OperationError> GetUserOrganizations(int userId);
         Result<bool, OperationError> CanActiveUserModifyCvr(Guid organizationUuid);
         Result<ResourcePermissionsResult, OperationError> GetPermissions(Guid organizationUuid);
+
+        public bool HasRole(int orgId, OrganizationRole role);
```

Comment:
I would prefer you create a GetPermissions method. The HasRole is fine as a private method used by the GetPermissions public method. 

The GetPermissions should return a GridPermissions model that is then mapped to the Response DTO you already created.

The issue with the current approach is that the Controller shouldn't contain any logic in it. It should only care about calling the service and handling the result. 

If you think a GetGridPermissions method doesn't belong in an OrganizationService feel free to create a new OrganizationGridService (then it would call the organizationService.HasRole to "gather" the permissions).

A good example is OrganizationUnitService::GetAccessRights